test/controllers/admin/bundles_controller_test.rb

```diff
@@ -8,6 +8,10 @@ class Admin::BundlesControllerTest < ActionController::TestCase
     FileUtils.rm_rf(APP_CONFIG.bundle_file_path)
   end
 
+  teardown do
```

Comment:
So it gets run after the last tests, so the library functions aren't screwed up after this test suite finishes